Valet.podspec

```diff
@@ -10,5 +10,16 @@ Pod::Spec.new do |s|
   s.public_header_files = 'Sources/*.h'
   s.frameworks = 'Security'
   s.ios.deployment_target = '9.0'
-  s.osx.deployment_target = '10.11'
+  s.macos.deployment_target = '10.11'
+
+  s.test_spec 'Tests' do |test_spec|
+    test_spec.ios.requires_app_host = true
+    test_spec.ios.source_files = 'Tests/**/*.{h,m,swift}'
+    test_spec.ios.exclude_files = 'Tests/MacTests.swift'
+    test_spec.macos.source_files = 'Tests/**/*.{h,m,swift}'
+    test_spec.pod_target_xcconfig = {
+      'SWIFT_OBJC_BRIDGING_HEADER' => '${PODS_TARGET_SRCROOT}/Tests/ValetTests-Bridging-Header.h',
+      'CLANG_WARN_UNGUARDED_AVAILABILITY' => 'YES'
```

Comment:
@dfed in Xcode 9 this is defaulting to `YES_AGGRESSIVE` I think but in your project you only have it to `YES` so you do not get any warnings during compilation of your test sources.

Reply:
Interesting. I guess we should do another pass to change our Xcode project to`YES_AGGRESSIVE`?

Reply:
yes we can do in a follow up. so we can fix the code and then remove this line from the podspec.